pkgs/build-support/writers/default.nix

```diff
@@ -301,6 +301,42 @@ let
   writePyPy3Bin = name:
     writePyPy3 "/bin/${name}";
 
+
+  makeFSharpWriter = { dotnet-sdk ? pkgs.dotnet-sdk, fsi-flags ? "", nugetDeps ? _: [] }: nameOrPath:
```

Comment:
maybe we should name `nugetDeps` `libraries` like in the other writers? could be nice to have a common interface

Reply:
Hi @Lassulus , thanks for taking a look.
Done.